Lib/ruby/director.swg

```diff
@@ -151,6 +151,8 @@ namespace Swig {
     const char *what() const throw() {
       return swig_msg.c_str();
     }
+
+    int status;
```

Comment:
This public member variable should be private with accessors added. Also, if I rerun the test-suite making this private it shows that it doesn't get used, so am wondering what is the purpose storing it?